Library/Homebrew/cmd/pull.rb

```diff
@@ -516,7 +516,11 @@ def verify_bintray_published(formulae_names)
           opoo "Cannot publish bottle: Failed reading info for formula #{f.full_name}"
           next
         end
-        bottle_info = jinfo.bottle_info(jinfo.bottle_tags.first)
+        if f.tap.remote.include?("Linuxbrew") && jinfo.bottle_tags.include?("x86_64_linux")
+          bottle_info = jinfo.bottle_info("x86_64_linux")
+        else
+          bottle_info = jinfo.bottle_info(jinfo.bottle_tags.first)
+        end
```

Comment:
In the interest of making it easier for everyone to bottle, I've decided to add this as another criteria for #18